docs/content/manual/pre-release/ha/backing-image-error-reporting-and-retry.md

```diff
@@ -10,12 +10,19 @@ title: Backing Image Error Reporting and Retry
 
 ## Backing image with sync failure
 1. Create a backing image. Then create and attach a volume using this backing image.
-2. Exec into the host of a worker node. remove the content of the backing image work directory then set the directory as immutable. This operation should fail one file of the backing image as well as keeping failing the following sync retries.
+2. Exec into the host of a worker node. remove the content of the backing image work directory then set the directory as immutable. 
+   This operation should fail one file of the backing image as well as keeping failing the following sync retries.
```

Comment:
What does failing one file mean here?

Reply:
Hi @innobead :
That's the origin test case description. I think I could modify it to say "**This should result in the failure of the sync operation and subsequent retries, as Longhorn will not be able to sync the data due to the immutable directory.**" What do you think?

Reply:
I think we can modify it into

2. Exec into one of the worker node, remove the files in that backing image directory and set the directory as immutable. 
   The removal of the files will trigger the sync process to sync backing image file from another worker node. Setting immutable to the directory will make the sync process and the following retry failed.